### DISCUSSION TITLE
关于“搜索功能无法使用”问题

### DISCUSSION THREAD

**Discussion by yikexiaoyao** [LIlGG/halo-theme-sakura#390] [Q&A]
### 目前使用的 Sakura 版本？

 2.0.0

### 目前使用的 halo 版本？

2.6.0

### 建议/问题

搜索功能无法使用，已经更换多个浏览器和设备测试
![Snipaste_2023-06-14_14-05-59](https://github.com/LIlGG/halo-theme-sakura/assets/36374114/61890df1-066e-45fc-bed4-39cc0db4afa3)

**Comment by yikexiaoyao** [comment]
![Snipaste_2023-06-14_14-10-41](https://github.com/LIlGG/halo-theme-sakura/assets/36374114/c567ba7a-2746-4ada-bcc1-003711c0d92c)

**Comment by LIlGG** [comment]
Beta.1 版本不支持搜索插件。 可以下载 [Beta.2](https://github.com/LIlGG/halo-theme-sakura/releases/tag/v2.0.0-beta.2) 版本。

**Comment by yikexiaoyao** [comment]
主题的问题？

**Comment by LIlGG** [comment]
> 主题的问题？

如果使用的是 Beta.1 版本，请查看 Release 声明。[搜索功能](https://github.com/LIlGG/halo-theme-sakura/releases#:~:text=Halo%202.x%20%E7%9B%AE%E5%89%8D%E9%9C%80%E8%A6%81%E4%BD%BF%E7%94%A8%E6%90%9C%E7%B4%A2%E6%8F%92%E4%BB%B6%20plugin%2Dsearch%2Dwidget%20%E6%9D%A5%E6%94%AF%E6%8C%81%E6%90%9C%E7%B4%A2%E3%80%82%E4%BD%86%E7%94%B1%E4%BA%8E%E4%B8%8E%E6%9C%AC%E4%B8%BB%E9%A2%98%E9%A3%8E%E6%A0%BC%E4%B8%8D%E7%AC%A6%EF%BC%8C%E7%AD%89%E5%BE%85%E5%9F%BA%E4%BA%8E%E8%B7%AF%E7%94%B1%E7%9A%84%E6%90%9C%E7%B4%A2%E5%8A%9F%E8%83%BD%E5%AE%8C%E6%88%90%E4%B9%8B%E5%90%8E%E5%86%8D%E8%BF%81%E7%A7%BB%E6%90%9C%E7%B4%A2%E5%8A%9F%E8%83%BD)
